enhancements/storage/csi-migration.md

```diff
@@ -0,0 +1,209 @@
+---
+title: Migration of in-tree volume plugins to CSI drivers
+authors:
+  - "@fbertina"
+reviewers:
+  - "@openshift/storage ”
+approvers:
+  - "@openshift/openshift-architects"
+creation-date: 2020-07-01
+last-updated: 2021-01-28
+status: provisional
+see-also:
+replaces:
+superseded-by:
+---
+
+# Migration of in-tree volume plugins to CSI drivers
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+We want to allow cluster administrators to seamlessly migrate in-tree volumes to their counterparts CSI drivers. It is important to achieve this goal before CSI Migration feature becomes GA in upstream.
+
+## Motivation
+
+CSI migration is going to be enabled by default as a beta feature in Kubernetes 1.21 (OCP 4.8). In Kubenertes 1.22 (OCP 4.9) the feature may become GA.
```

Comment:
A little confused by this first sentence since looking at the KEP issue it's already beta since at least 1.20?